doc/api/fs.md

```diff
@@ -1007,6 +1007,39 @@ property indicating whether parent directories should be created. Calling
 `fsPromises.mkdir()` when `path` is a directory that exists results in a
 rejection only when `recursive` is false.
 
+```mjs
+import { mkdir } from 'fs/promises';
+
+try {
+  const path = new URL('./test/project', import.meta.url);
```

Comment:
I'm not sure using URL here makes this example clearer.

Reply:
would you have a different/preferred ESM way to do a relative path that can be reliably copy/pasted?

Reply:
Might be good to name it `projectPath` or something like that. In our docs at least, it might be best to reserve `path` for references to the `path` module.

Reply:
I think `new URL(somePath, import.meta.url)` is pretty standard for ESM at this point. but maybe someone else can say with more certainty if there is a more reader-friendly option.  @nodejs/modules @aduh95

Reply:
> I think `new URL(somePath, import.meta.url)` is pretty standard for ESM at this point.

Agreed, I can’t think of a better/more idiomatic way. Also agree with changing `const path` to `const projectPath` or some other name.

Reply:
> Also agree with changing const path to const projectPath or some other name.

I suggest naming it `projectFolder`, so we're sure it won't be confused with `node:path`.

Reply:
Also adding a `/` at the end of the url is good practice so the URL object can serve as base for sub-URLs.
```suggestion
  const projectFolder = new URL('./test/project/', import.meta.url);
```

Reply:
> Also adding a `/` at the end of the url is good practice so the URL object can serve as base for sub-URLs.

I removed `author ready` because of this one suggested change. But other than that, I think this is ready to land.